bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestBigtableTable.java

```diff
@@ -117,7 +117,13 @@ public void setup() throws IOException {
     when(mockConnection.getBigtableSettings()).thenReturn(settings);
     when(mockBigtableApi.getDataClient()).thenReturn(mockBigtableDataClient);
     when(mockBigtableDataClient.readRows(isA(Query.class))).thenReturn(mockResultScanner);
-    table = new AbstractBigtableTable(mockConnection, hbaseAdapter) {};
+    table =
+        new AbstractBigtableTable(mockConnection, hbaseAdapter) {
+          @Override
+          public void mutateRow(RowMutations rowMutations) throws IOException {
+            mutateRowVoid(rowMutations);
```

Comment:
How is table used? does the test need to check the hbase version and call different method?